aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -605,4 +669,62 @@ module aptos_framework::account {
         let pk = x"0000000000000000000000000000000000000000000000000000000000000000";
         rotate_authentication_key_ed25519(&alice, test_signature, test_signature, pk, pk);
     }
+
+    #[test(bob = @0x345)]
+    public entry fun test_valid_offer_rotation_capability(bob: signer) acquires Account {
```

Comment:
this test is a smaller version of `test_valid_accept_rotation_capability`, let's delete it.